src/manifest.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+const Address = require('./address')
+const { Block } = require('multformats/block')
+const { sha256: hasher } = require('multiformats/hashes/sha2')
+const mhtype = 'sha2-256'
+const codec = require('@ipld/dag-cbor')
+const defaultTimeout = 30000
+const proxyHandler = {
+  get: (target, prop) => prop.startsWith('_') ? undefined : target[prop]
+}
+
+class Manifest {
+  constructor ({ _block }) {
+    Object.keys(_block.value).forEach((k) => { this[k] = _block.value[k] })
+
+    this._block = _block
+    this._address = Address.asAddress({ cid: this.cid }, true)
+  }
+
+  static async create (orbitdb, { AccessControllers, name, type, ...options } = {}) {
+    const value = { name, type }
+    const accessControllerAddress = await AccessControllers.create(
+      orbitdb,
+      options.accessController?.type || 'ipfs',
+      options.accessController || {}
+    )
```

Comment:
it may be best to have the accessControllers use the `Manifest.create` method to create their manifest eventually.
As part of 1.0, I'd like to see the orbit-db accessController to not use the ipfs accessController, creating another manifest within a manifest within a manifest. This would make it easier to tell that: if we have the manifest and the accessController options resolved, then we have everything we need to open and run the store.